tests/functional/aws-node-sdk/test/multipleBackend/get/getAzure.js

```diff
@@ -53,7 +53,10 @@ function testSuite() {
             });
         });
         keys.forEach(key => {
-            describe(`${key.describe} size`, () => {
+            describe(`${key.describe} size`, function fn() {
```

Comment:
I just noticed this. Having describe in a forEach is a bad idea as it's not synchronous, meaning two tests run simultaneously and if they happen to use the same bucket name or key name, they are bound to have a collision. We have to clean this up later.